pixeltable/exprs/column_ref.py

```diff
@@ -37,6 +37,7 @@ class ColumnRef(Expr):
     """
 
     col: catalog.Column
+    reference_tbl: catalog.TableVersionPath
```

Comment:
Add an explanation of why this is needed to the class comment.

Reply:
Done.